### DISCUSSION TITLE
Won't ever release Airyx OS with arm architecture support for arm laptops?

### DISCUSSION THREAD

**Discussion by muhammadbahaa2001** [ravynsoft/ravynos#175] [General]
Most laptops are based on x86_64 architecture but won't we see arm architecture support for arm laptops? and if that happened won't Airyx OS able to run iOS/iPad apps? (After Airyx OS support Mac apps)

**Comment by mszoek** [comment]
I hope to support ARM eventually, but it is not a priority right now. There's a huge amount of work just to do x86-64 well so that is the first focus area. It might happen sooner if someone volunteers to help do & maintain the ARM port. 😺

**Comment by davidchisnall** [comment]
Note that FreeBSD already has tier-1 support for AArch64 and the Objective-C runtime has all of the assembly parts working, so this may be less work for a volunteer than they might imagine.

**Comment by RangerRick** [comment]
I've been working on getting it going; I can get everything built but haven't managed to do a clean bootstrap yet.
